__tests__/tests/components/CommentInput.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Platform} from 'react-native';
+
+import {CommentInput} from 'components';
+
+describe('<CommentInput />', () => {
+
+  const defaultProps = {
+    users: [],
+    userHasPushPermission: true,
+    issueLocked: false,
+    locale: '',
+    onSubmit: () => {},
+  };
+
+  it('should render styled TextInput and styled TouchableOpacity if user has post permissions and issue is not locked', () => {
+
+    const wrapper = shallow(
+      <CommentInput {...defaultProps}/>
+    );
+
+    expect(wrapper.find('Styled(TextInput)').length).toEqual(1);
+    expect(wrapper.find('Styled(TouchableOpacity)').length).toEqual(1);
+    expect(wrapper.find('Styled(PostButtonIcon)').length).toEqual(1);
+    expect(wrapper.find('Styled(Text)').length).toEqual(0);
```

Comment:
The description didn't mention  "Styled(PostButtonIcon)" and "Styled(Text)". They are what I mean "more components".